_tools/check_docs.ts

```diff
@@ -23,131 +23,38 @@ import {
-  "../encoding/unstable_hex.ts",
-  "../encoding/unstable_base64_stream.ts",
-  "../encoding/unstable_base32_stream.ts",
-  "../encoding/unstable_hex_stream.ts",
-  "../expect/mod.ts",
-  "../fmt/bytes.ts",
-  "../fmt/colors.ts",
-  "../fmt/duration.ts",
-  "../fmt/printf.ts",
-  "../front_matter/mod.ts",
-  "../front_matter/unstable_yaml.ts",
-  "../fs/mod.ts",
-  "../fs/unstable_chmod.ts",
-  "../fs/unstable_copy_file.ts",
-  "../fs/unstable_link.ts",
-  "../fs/unstable_lstat.ts",
-  "../fs/unstable_make_temp_dir.ts",
-  "../fs/unstable_make_temp_file.ts",
-  "../fs/unstable_mkdir.ts",
-  "../fs/unstable_read_dir.ts",
-  "../fs/unstable_read_file.ts",
-  "../fs/unstable_read_link.ts",
-  "../fs/unstable_read_text_file.ts",
-  "../fs/unstable_real_path.ts",
-  "../fs/unstable_remove.ts",
-  "../fs/unstable_rename.ts",
-  "../fs/unstable_stat.ts",
-  "../fs/unstable_symlink.ts",
-  "../fs/unstable_truncate.ts",
-  "../fs/unstable_types.ts",
-  "../fs/unstable_umask.ts",
-  "../fs/unstable_utime.ts",
-  "../fs/unstable_write_file.ts",
-  "../fs/unstable_write_text_file.ts",
-  "../html/mod.ts",
-  "../html/unstable_is_valid_custom_element_name.ts",
-  "../http/mod.ts",
-  "../http/unstable_header.ts",
-  "../http/unstable_method.ts",
-  "../http/unstable_signed_cookie.ts",
-  "../ini/mod.ts",
-  "../internal/mod.ts",
-  "../io/mod.ts",
-  "../json/mod.ts",
-  "../jsonc/mod.ts",
-  "../log/base_handler.ts",
-  "../log/file_handler.ts",
-  "../log/warn.ts",
-  "../log/critical.ts",
-  "../log/debug.ts",
-  "../log/error.ts",
-  "../log/info.ts",
-  "../log/console_handler.ts",
-  "../log/formatters.ts",
-  "../log/get_logger.ts",
-  "../log/logger.ts",
-  "../media_types/mod.ts",
-  "../msgpack/mod.ts",
-  "../net/mod.ts",
-  "../net/unstable_get_network_address.ts",
-  "../path/mod.ts",
-  "../path/unstable_basename.ts",
-  "../path/unstable_dirname.ts",
-  "../path/unstable_extname.ts",
-  "../path/unstable_join.ts",
-  "../path/unstable_normalize.ts",
-  "../path/posix/mod.ts",
-  "../path/windows/mod.ts",
-  "../random/mod.ts",
-  "../regexp/mod.ts",
-  "../semver/mod.ts",
-  "../streams/mod.ts",
-  "../streams/unstable_fixed_chunk_stream.ts",
-  "../streams/unstable_to_lines.ts",
-  "../streams/unstable_to_bytes.ts",
-  "../tar/mod.ts",
-  "../text/mod.ts",
-  "../text/unstable_slugify.ts",
-  "../text/unstable_to_constant_case.ts",
-  "../testing/bdd.ts",
-  "../testing/mock.ts",
-  "../testing/snapshot.ts",
-  "../testing/time.ts",
-  "../testing/types.ts",
-  "../toml/mod.ts",
-  "../ulid/mod.ts",
-  "../uuid/mod.ts",
-  "../uuid/unstable_v7.ts",
-  "../webgpu/mod.ts",
-  "../yaml/mod.ts",
-] as const;
+const STABLE_VERSION = parse("1.0.0");
+const ROOT = new URL("../", import.meta.url);
+const ENTRY_POINT_URLS = [];
+for await (
+  const { path } of walk(ROOT, { exts: [".json"], match: [/deno.json$/] })
+) {
+  if (isTestFile(path)) continue;
+  const { version, exports } = JSON.parse(await Deno.readTextFile(path));
```

Comment:
Another option is to dynamically import using the `json` assertion type.

Reply:
That might be a possibility.
Sidenote: `deno.jsonc` is also a valid with deno, so we might want to support `jsonc` to make this fool-proof in case some std module with use a `deno.jsonc` in the future.

Reply:
IMHO, it's a little cleaner. Also, I wouldn't worry about supporting something that isn't currently used. Because it's also possible that `deno.jsonc` files will never be used.

Reply:
Ok for now. But this might unintentionally break things in the future or files might fall outside without throwing, so I think we should generally cover all possible use cases especially for such checks.